src/docs/project_goals.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Project Goals
+---
+
+# Strategic Goals of Eclipse Theia
+
+This section describes the overall goals of the Eclipse Theia project.
+
+## Eclipse Theia mission statement: A framework for building tools and IDEs based on web technologies
```

Comment:
Should this header be removed? It currently looks odd since it has no content:

<img width="1043" alt="Screen Shot 2022-03-15 at 5 37 34 PM" src="https://user-images.githubusercontent.com/40359487/158477435-f0001b99-1b8c-4d39-99c8-7e247adf2b28.png">

Reply:
It is the mission statement, so I wanted it to be big

Reply:
The problem is that it has the same header style as other subtitles so it seems out of place.

Reply:
I agree with Vince, we should probably make it stand out a bit. Maybe add a `font-weight: bold` to make it clear, that it's not an actual header?

Reply:
done